website/blog/CONTRIBUTING.md

```diff
@@ -0,0 +1,298 @@
+1. Update `src/revisions.ts` with the found revision number.
+1. Update `versions.js` with the new Chromium-to-Puppeteer version mapping.
+1. Run `npm run ensure-correct-devtools-protocol-revision`.
+   If it fails, update `package.json` with the expected `devtools-protocol` version.
+1. Run `npm run tsc` and `npm install`.
+1. Run `npm run unit` and ensure that all tests pass. If a test fails, [bisect](#bisecting-upstream-changes) the upstream cause of the failure, and either update the test expectations accordingly (if it was an intended change) or work around the changes in Puppeteer (if it’s not desirable to change Puppeteer’s observable behavior).
+1. Commit and push your changes and open a pull request.
+   The commit message must contain the version in `Chromium <version> (<revision>)` format to ensure that [pptr.dev](https://pptr.dev/) can parse it correctly, e.g. `'feat(chromium): roll to Chromium 90.0.4427.0 (r856583)'`.
+
+### Bisecting upstream changes
+
+Sometimes, performing a Chromium roll causes tests to fail. To figure out the cause, you need to bisect Chromium revisions to figure out the earliest possible revision that changed the behavior. The script in `utils/bisect.js` can be helpful here. Given a Node.js script that calls `process.exit(1)` for bad revisions, run this from the Puppeteer repository’s root directory:
+
+```sh
+node utils/bisect.js --good 686378 --bad 706915 script.js
+```
+
+## Releasing to npm
+
+Releasing to npm consists of the following phases:
+
+1. Source Code: mark a release.
+    1. Run `npm run release`. (This automatically bumps the version number in `package.json`, populates the changelog, updates the docs, and creates a Git commit for the next step.)
+    1. Send a PR for the commit created in the previous step.
+    1. Make sure the PR passes **all checks**.
+        - **WHY**: there are linters in place that help to avoid unnecessary errors, e.g. [like this](https://github.com/puppeteer/puppeteer/pull/2446)
+    1. Merge the PR.
+    1. Once merged, publish the release notes from `CHANGELOG.md` using [GitHub’s “draft new release tag” option](https://github.com/puppeteer/puppeteer/releases/new).
+        - **NOTE**: tag names are prefixed with `'v'`, e.g. for version `1.4.0` the tag is `v1.4.0`.
+    1. As soon as the Git tag is created by completing the previous step, our CI automatically `npm publish`es the new releases for both the `puppeteer` and `puppeteer-core` packages.
+1. Source Code: mark post-release.
+    1. Bump `package.json` version to the `-post` version, run `npm run doc` to update the “released APIs” section at the top of `docs/api.md` accordingly, and send a PR titled `'chore: bump version to vXXX.YYY.ZZZ-post'` ([example](https://github.com/puppeteer/puppeteer/pull/6808))
+        - **NOTE**: no other commits should be landed in-between release commit and bump commit.
```

Comment:
for now, let's just make the website CONTRIBUTING document link to the GitHub page, to avoid duplicating the content. So this should just have something like:

```
Head to GitHub to view our CONTRIBUTING.md document
```

We'll tidy this up and make it nicer in the future, but we probably don't want to have it duplicated.

Reply:
Sure, will update the COntributing.md